src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+
+  if (days < 3) {
+    return dailyRent * days;
+  } else if (days < 7) {
+    return (dailyRent * days) - 20;
+  } else {
```

Comment:
I like the idea of declaring variable 'dailyRent' and the use of if/else if/else statement is perfect. One recommandation from myside you can remove the last else statement and can only return "(dailyRent * days) - 50" this. It will works the same.